controllers/tempest_controller.go

```diff
@@ -492,6 +492,7 @@ func (r *TempestReconciler) setTempestConfigVars(envVars map[string]string,
 		envVars["TEMPEST_EXTRA_IMAGES_ID"] += extraImageDict.ID + ","
 		envVars["TEMPEST_EXTRA_IMAGES_NAME"] += extraImageDict.Name + ","
 		envVars["TEMPEST_EXTRA_IMAGES_DISK_FORMAT"] += extraImageDict.DiskFormat + ","
+		envVars["TEMPEST_IMAGE_CREATE_TIMEOUT"] += r.GetDefaultInt(extraImageDict.ImageCreationTimeout) + ","
```

Comment:
The env variable name should be changed if we go with [1]
`TEMPEST_IMAGE_CREATE_TIMEOUT` -> `TEMPEST_EXTRA_IMAGES_CREATE_TIMEOUT`

[1] https://github.com/openstack-k8s-operators/tcib/pull/199